lang/en/settings.php

```diff
@@ -7,10 +7,11 @@
  */
  
 // for the configuration manager
-$lang['style']            = 'Style of pagelist tables';
+$lang['style']            = 'Style of pagelist';
 $lang['style_o_default']  = 'default';
 $lang['style_o_table']    = 'table';
-$lang['style_o_list']     = 'list';
+$lang['style_o_list']     = 'table/list';
```

Comment:
Is list  also a table? 
My tests with only pagelist plugin does show it as list.
The code suggests it displays only pagenames. https://github.com/dokufreaks/plugin-pagelist/blob/master/helper.php#L130
So what about `List` and `Simple List (only page names)`